life_expectancy/tests/test_data_load_save.py

```diff
@@ -20,3 +28,47 @@ def test_save_data(pt_life_expectancy_expected):
             pt_life_expectancy_expected, "to_csv") as mock_to_csv:
         save_data(pt_life_expectancy_expected)
         mock_to_csv.assert_called_once()
+
+
+def test_tsv_load_strategy_default_path():
+    strategy = TSVLoadStrategy()
+    with mock.patch('pandas.read_csv') as mock_read_csv:
+        strategy.load_data()
+        expected_path = join(get_current_directory_full_path(), "data", "eu_life_expectancy_raw.tsv")
+        mock_read_csv.assert_called_with(expected_path, sep="\t", header=0)
+
+
+def test_default_json_file_path():
+    strategy = JSONLoadStrategy()
+    data = strategy.load_data()
+    assert isinstance(data, pd.DataFrame)
+    # You can add more assertions based on the expected content of the default file.
+
+
+def test_create_strategy_for_tsv():
```

Comment:
i don't think this is a necessary test, but we could ask Fernando for help here

Reply:
I think it's fair game to test the factory function. You could simplify the test using parameterized:

```python
@pytest.mark.parametrize(
  "suffix, expected_strategy", 
  [
    (".tsv", TSVLoadStrategy), 
    (".json", JSONLoadStrateg)
  ]
)
def test_create_strategy(suffix, expected_strategy):
    strategy = create_strategy(suffix)
    assert isinstance(strategy, expected_strategy)
```